frontend/app/(footer)/user/[id]/page.tsx

```
@@ -130,55 +114,32 @@ export default function UserProfilePage({ params }: any) {
-            <p className="md:pt-8 pb-6 font-light">
-              <RichTextEditorContent description={publicUserProfile.bio || "No bio provided."} />
-            </p>
+            <div className="md:pt-8 pb-6">
+              <div className={`font-light ${isBioExpanded ? "" : "line-clamp-4 md:line-clamp-5"}`}>
+                <RichTextEditorContent description={publicUserProfile.bio || "No bio provided."} />
+              </div>
+              {publicUserProfile.bio && publicUserProfile.bio.length > 100 && (
+                <button
+                  onClick={() => setIsBioExpanded(!isBioExpanded)}
+                  className="text-sm text-core-text hover:underline mt-2 font-medium"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)